cdk/lib/registration.ts

```diff
@@ -17,43 +14,8 @@ export class Registration extends GuStack {
 		// Until this project has been fully migrated to GuCDK you should update the 'old' infrastructure by modifying
 		// the YAML file and then re-running the snapshot tests to confirm that the changes are being pulled through by
 		// CDK
-		const cfnTemplate = new CfnInclude(this, 'YamlTemplate', {
+		new CfnInclude(this, 'YamlTemplate', {
```

Comment:
This seems a bit pointless now that we're not actually provisioning any resources via GuCDK, but I don't think it's worth rolling back this wiring (introduced via https://github.com/guardian/mobile-n10n/pull/831) as it will [slightly reduce the amount of work required when we decide to migrate this to a pattern](https://github.com/guardian/cdk/blob/main/docs/migration-guide.md#migration-to-cdk).

Reply:
Agree!